src/provider/lowebutil.js

```diff
@@ -11,7 +11,7 @@ export function getParameterByName(name, url) {
 }
 
 export function isElectron() {
-  return window && window.process && window.process.platform;
+  return window.api?.platform;
```

Comment:
bug here: it will return platform name in electron environment, not a boolean value